src/Cosmo.php

```diff
@@ -256,18 +256,29 @@ public function quote(string $quote): string
     /**
      * @param float $value
      * @param string $currency The 3-letter ISO 4217 currency code indicating the currency to use.
+     * @param int|null $precision The needed number of decimals digits 
      * @param string|null $pattern
      * @return string
      * @throws Exception
      */
-    public function money(float $value, string $currency = null, string $pattern = ''): string
+    public function money(float $value, string $currency = null, int $precision= -1,  string $pattern = ''): string
```

Comment:
Just wondering isn't better if we make `$percision` nullable instead of using `-1` as default (` ?int $precision`)? What could be pros and cons? From the API perspective, I believe null is more intuitive.